core/platform/datastore/gae_datastore_services.py

```diff
@@ -216,14 +216,14 @@ def fetch_multiple_entities_by_ids_and_models(ids_and_models):
     return all_models_grouped_by_model_type
 
 
-def make_pseudo_random_hr_consistency_policy():
+def instantaneous_global_consistency_policy():
```

Comment:
Method/function names should always start with a verb.

Why are we renaming this, btw?

Reply:
OK, have prefixed it with `get_`.

Because the name implied that the policy was "psuedo random", when in fact we explicitly set it to be _not_ random, so I wanted the name to be closer to what's actually being returned.